client/js/libs/handlebars/diff.js

```diff
@@ -1,5 +1,5 @@
 var diff;
```

Comment:
Do you mind looking why we have this global variable, and remove it if possible?